webapi/src/main/scala/org/knora/webapi/messages/v2/responder/ontologymessages/KnoraApiV2Simple.scala

```diff
@@ -757,11 +285,11 @@ object KnoraApiV2Simple {
         )
     )
 
-    val HasDDDFile: ReadPropertyInfoV2 = makeProperty(
+    private val HasDDDFile: ReadPropertyInfoV2 = makeProperty(
```

Comment:
Couldn't this not even be made more compact? The systematic difference is that we expose a `knora-base` property pointing to  a `FileValue` as a `knora-api` property with an object type `File` to the outside. This applies to all internal properties pointing to file values. 

Do you think we could reuse the internal labels and comments? Then they would not have to be repeated. Of course if we say in the comment that the property points to a value then this does not work.